Main.hs

```diff
@@ -11,144 +11,156 @@ import qualified Data.List as L
 import Text.Printf (printf)
 
 multByIndex :: [Int] -> [Int]
-multByIndex = undefined
+multByIndex xs = [x * i | (x, i) <- zip xs [0 ..]]
 
 powerByIndex :: [Int] -> [Int]
-powerByIndex = undefined
+powerByIndex xs = [x ^ i | (x, i) <- zip xs [0 ..]]
```

Comment:
same)

Reply:
I did the changes you requested in main.hs. is it ok now?